xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/general.less

```diff
@@ -183,7 +183,8 @@ body {
 
       /* Remove the underlining from links with the btn class.
        Those are already styled differently from their neighboring text. */
-      & a.btn {
+      & a.btn,
+      & a.button {
```

Comment:
I don't understand this change, you didn't introduced a button class here, but you replaced a link to a button, so the selector should be just button no?

Reply:
![image](https://github.com/user-attachments/assets/e48e9d7a-65bd-41ca-be26-316584532580)

When checking out the displays of "button like links", I stumbled upon this one. It's underlined but there's no need for it to be underlined. See the third item of the PR description in the opening message.